### PR TITLE
Add pre-built image for grpc-dotnet

### PR DESCRIPTION
**Pull request by jtattermusch**
Adds the Dockerfile for "prebuilt" version of grpc-dotnet qps worker.

### CODE REVIEW

**Comment by jtattermusch** [comment]
CC @JamesNK

File: containers/pre_built_workers/dotnet/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+RUN git submodule update --init
+
+# Save commit sha for debug use
+RUN echo 'COMMIT SHA' > GRPC_GIT_COMMIT.txt
+RUN git rev-parse $GITREF >> GRPC_GIT_COMMIT.txt
+
+# Build standalone build of qps worker, put output into qps_worker directory.
+RUN dotnet publish -c Release -o qps_worker perf/benchmarkapps/QpsWorker/
+
+# Note that the libgrpc_csharp_ext.so library built by the build image
```
Comment (JamesNK) [inline containers/pre_built_workers/dotnet/Dockerfile:41]: Is this comment relevant for grpc-dotnet?

**Review by paulosjca — COMMENTED** [review]
I think it makes more sense to wait until the examples are generated from templates before checking this in. This change is not required for continuous integration, because continuous integration uses the templates in grpc/grpc, but does not use the examples.

File: containers/pre_built_workers/dotnet/Dockerfile (hunk 1 above)
Comment (jtattermusch) [reply containers/pre_built_workers/dotnet/Dockerfile]: Good point. Updated.

**Comment by jtattermusch** [comment]
I updated the dockerfile according to comments. I tested building the image locally off of the master branch of grpc-dotnet.
Since the dockerfile is required to be able to run the benchmarks on the CI (even for just experiments), I'm going to merge this PR and address TODOs in a followup PR.

TODO: add support for grpc-dotnet in non-pre-built image (not immediately necessary for the CI runs, since they only use pre-built images)

TODO: regenerate the examples based on https://github.com/grpc/grpc/blob/master/tools/run_tests/performance/README.md#generating-load-test-examples

**Pull request merged**